app/views/organisation/consultations/show.html.slim

```diff
@@ -28,14 +53,25 @@
                         .dropdown-with-icon
                           p Delete
                           .pull-right.fas.fa-trash-alt.pointer.fa-icon-color
-  .admin-list-bg
+  .admin-list-bg class="#{@consultation.expired? ? 'consultation-deadline-top' : ''}"
     .p-4
       .admin-index-title
         | Basic Information
       .admin-details-box
         = display_details("show-page-parent-details", "show-page-label-details", "Consultation title", "show-page-input-details", "#{@consultation&.title || '-'}")
         = display_details("show-page-parent-details", "show-page-label-details", "Feedback Email", "show-page-input-details", "#{@consultation&.consultation_feedback_email || '-'}")
         = display_details("show-page-parent-details", "show-page-label-details", "Consultation PDF", "show-page-input-details", "#{@consultation&.url}")
+        .show-page-parent-details
+          .show-page-label-details Status
+          .show-page-input-details
+            div class="#{@consultation.status}"
+              .text-color.m-1
+                - if @consultation.status == "submitted"
```

Comment:
@BALAJI-BALA - can we use `consultation.submitted`?

Also, make this into a view helper